ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -42,7 +46,8 @@ export class MigrationProjectService extends AbstractService {
 
         if (event.isTypeOf(NewExecutionStartedEvent)) {
             monitoredProject.executions.push((<NewExecutionStartedEvent>event).execution);
-        } else if (event.isTypeOf(ExecutionUpdatedEvent)) {
+        }
+        else if (event.isTypeOf(ExecutionUpdatedEvent)) {
```

Comment:
Please don't just switch to this format. It isn't the conventions that we have been using everywhere else.